nevergrad/optimization/lamcts/utils.py

```diff
@@ -0,0 +1,43 @@
+###############################################################################
```

Comment:
this is an issue, this license is too restrictive to incorporate inside nevergrad